protocols/gossipsub/src/lib.rs

```diff
@@ -156,18 +156,73 @@ mod types;
 
 mod rpc_proto;
 
-pub use self::behaviour::{Gossipsub, GossipsubEvent, MessageAuthenticity};
+pub use self::behaviour::{Behaviour, Event, MessageAuthenticity};
 pub use self::transform::{DataTransform, IdentityTransform};
 
-pub use self::config::{GossipsubConfig, GossipsubConfigBuilder, GossipsubVersion, ValidationMode};
 pub use self::peer_score::{
     score_parameter_decay, score_parameter_decay_with_base, PeerScoreParams, PeerScoreThresholds,
     TopicScoreParams,
 };
 pub use self::topic::{Hasher, Topic, TopicHash};
-pub use self::types::{
-    FastMessageId, GossipsubMessage, GossipsubRpc, MessageAcceptance, MessageId,
-    RawGossipsubMessage,
-};
+pub use self::types::{FastMessageId, Message, MessageAcceptance, MessageId, RawMessage, Rpc};
+
+pub use self::config::{Config, ConfigBuilder, ValidationMode, Version};
+
```

Comment:
```suggestion
pub use self::config::{Config, ConfigBuilder, ValidationMode, Version};
```

Let's not use more whitespace than necessary :)

Reply:
:heavy_check_mark: